docs/upgrading-fcos.md

```diff
@@ -21,6 +21,115 @@ There are no breaking changes between versions 1.4.0 and 1.5.0 of the `fcos` con
 
 The following is a list of notable new features.
 
+### Offline Tang provisioning
+
+The `tang` section gained a new `advertisement` field. This can be specified in both the `storage` and/or the `boot_device`. If specified, Ignition will use it to provision the Tang server binding rather than fetching the advertisement from the server at runtime. This allows the server to be unavailable at provisioning time. The advertisement can be obtained from the server with `curl http://tang.example.com/adv`.
+
+<!-- butane-config -->
+```yaml
+variant: fcos
+version: 1.5.0
+boot_device:
+  layout: ppc64le
+  mirror:
+    devices:
+      - /dev/sda
+      - /dev/sdb
+  luks:
+    tang:
+      - url: https://tang.example.com
+        thumbprint: REPLACE-THIS-WITH-YOUR-TANG-THUMBPRINT
+        advertisement: "{\"payload\": \"...\", \"protected\": \"...\", \"signature\": \"...\"}"
+storage:
+  luks:
+    - name: luks-tang
+      device: /dev/sdc
+      clevis:
+        tang:
+          - url: https://tang.example.com
+            thumbprint: REPLACE-THIS-WITH-YOUR-TANG-THUMBPRINT
+            advertisement: "{\"payload\": \"...\", \"protected\": \"...\", \"signature\": \"...\"}"
+```
+
+### LUKS discard
+
+The `luks` section gained a new `discard` field. This can be specified in both the `storage` and/or the `boot_device`. If specified and true, the LUKS volume will issue discard commands to the underlying block device when blocks are freed. This improves performance and device longevity on SSDs and space utilization on thinly provisioned SAN devices, but leaks information about which disk blocks contain data.
+
+<!-- butane-config -->
+```yaml
+variant: fcos
+version: 1.5.0
+boot_device:
+  layout: ppc64le
+  mirror:
+    devices:
+      - /dev/sda
+      - /dev/sdb
+  luks:
+      discard: true
```

Comment:
Overindented.  Also, we need to configure some encryption here, probably `tpm2: true` for simplicity.

Reply:
Not fixed.

Reply:
Sorry, I could have swore I fixed that...

Reply:
Nope, the indentation hasn't changed.